Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.yml

```
@@ -762,17 +774,857 @@ script:
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.CreatedBy.Application.DisplayName
+      description: 'Name of the application who created the eDiscovery search.'
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.CreatedBy.Application.ID
+      description: 'ID of the application who created the eDiscovery search.'
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.CreatedBy.User.DisplayName
+      description: 'Name of the user who created the eDiscovery search.'
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.CreatedBy.User.ID
+      description: 'ID of the user who created the eDiscovery search.'
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.CreatedBy.User.UserPrincipalName
+      description: 'Internet-style login name of the user who created the eDiscovery search.'
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.CreatedDateTime
+      description: 'The date and time the eDiscovery search was created.'
+      type: Date
+    - contextPath: MsGraph.eDiscoverySearch.DataSourceScopes
+      description: 'When specified, the collection will span across a service for an entire workload. Possible values are: none, allTenantMailboxes, allTenantSites, allCaseCustodians, allCaseNoncustodialDataSources.'
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.Description
+      description: 'The description of the eDiscovery search.'
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.DisplayName
+      description: 'The display name of the eDiscovery search.'
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.LastModifiedDateTime
+      description: ''
+      type: Date
+    - contextPath: MsGraph.eDiscoverySearch.SearchId
+      description: 'The ID for the eDiscovery search.'
+      type: String
+  - arguments:
+    - description: The id of the eDiscovery case.
+      isArray: false
+      name: case_id
+      required: true
+    - description: The id of the eDiscovery search.
+      isArray: false
+      name: search_id
+      required: true
+    - description: The display name of the search.
+      isArray: false
+      name: display_name
+      required: true
+    - description: The description of the search.
+      isArray: false
+      name: description
+      required: false
+    - description: The query string used for the search. The query string in KQL (Keyword Query Language) format. For details, see Keyword queries and search conditions for Content Search and eDiscovery. You can refine searches by using fields paired with values; for example, subject:"Quarterly Financials" AND Date>=06/01/2016 AND Date<=07/01/2016.
```

```suggestion
    - description: The query string used for the search. The format of the query string is KQL (Keyword Query Language). For details, see Keyword queries and search conditions for Content Search and eDiscovery. You can refine searches by using fields paired with values, for example, subject:"Quarterly Financials" AND Date>=06/01/2016 AND Date<=07/01/2016.
```
Where is this:  Keyword queries and search conditions for Content Search and eDiscovery?  Need a link or remove.